### DISCUSSION TITLE
Is it possible to skip a page if it has no content?

### DISCUSSION THREAD

**Discussion by bolangu** [QuestPDF/QuestPDF#582] [Q&A]
Because the source data is dynamic, sometimes a page's content is empty and the PDF document shows an empty page with only header and footer. It would be ideal to skip such pages. Any thoughts?

**Comment by MarcinZiabek** [comment]
Would you please elaborate more about your use case? Best, provide some visual examples, and code if possible.

**Comment by bolangu** [comment]
I figured it out. I removed the pagebreak command when there is no content on a page. That did the trick.
